### PR TITLE
MFA

### PR DESCRIPTION
**Pull request by creeper00**

### CODE REVIEW

**Review by kooyunmo — CHANGES_REQUESTED** [review]
I just left one minor comment! Everything else LGTM!

File: pfcli/main.py [hunk 1]
```diff
@@ -87,22 +87,17 @@ def login(
...
-        typer.echo("| |  |  __/ |  | ||  __|| | (_) | V  V / ")
-        typer.echo("|_|   \___|_|  |_||_|   |_|\___/ \_/\_/  ")
-        typer.echo("\n\n")
-    except HTTPError:
-        secho_error_and_exit("Login failed... Please check your username and password.")
-
+    resp = r.json()
+    if "code" in resp and resp["code"] == "mfa_required":
+        mfa_token = resp["mfa_token"]
+        client: UserClientService = build_client(ServiceType.MFA)
```
[the remark this answers is not in the document]
Comment (creeper00) [reply pfcli/main.py]: Oh, thanks for finding. My mistake

**Comment by kooyunmo** [comment]
@creeper00 Can you please resolve the conflict before merge?

**Review by kooyunmo — APPROVED** [review]
LGTM!

**Pull request merged**